src/org/labkey/test/util/CspLogUtil.java

```diff
@@ -38,7 +38,7 @@ private CspLogUtil() { }
 
     public static void checkNewCspWarnings(ArtifactCollector artifactCollector)
     {
-        if (TestProperties.isCspCheckSkipped() || TestProperties.isServerRemote() || missingLog)
+        if (TestProperties.isServerRemote() || missingLog)
```

Comment:
I trust this is intentional, just curious why the `isCspCheckSkipped` check is no longer needed.

Reply:
This way it will still log the CSP violations in the test log, even if the check is disabled